okapi/services/caches/reports/reports/docs.xml

```diff
@@ -0,0 +1,15 @@
+<xml>
```

Comment:
I believe none of these will work on OCPL, right? So we need some of these newly introduced "tags" here.

Reply:
It all works at both OCPL and OCPL